client/src/globals/UserContext.js

```diff
@@ -8,6 +8,22 @@ export const UserContext = createContext({
 export const UserContextProvider = (props) => {
     const [userId, setUserId] = useState("");
     const value = { userId, setUserId };
+    useEffect(() => {
+        let status;
+        fetch("/getcookie", {
+            method: "GET",
+            headers: {
+                "Content-Type": "application/json"
+            }
```

Comment:
don't forget the credentials:"include" to be able to send cookies to the backend

Reply:
Thanks for reminder, and this branch was actually built up on the branch of userContext.